examples/graphics/area-lights.html

```diff
@@ -36,9 +37,9 @@
             material.useMetalness = true;
 
             if (assetManifest) {
```

Comment:
Will this always be true?

Reply:
the function is called with null in some cases, where dont want to use loaded textures